content/pages/changelog/index.md

```diff
@@ -13,6 +13,213 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-06-11 / New enterprise events for tasks
+
+Starting today, the [enterprise event stream](g://events/for-enterprise/) will
+begin producing new task and task assignment events, and some existing task
+events will return additional fields.
+
+### New task and task assignment events
+
+#### Task update
+
+```js
+      {
+            "source": {
+                "item_type": "file",
+                "item_id": "123",
+                "item_name": "github_PNG20 (3) (2).png",
+                "parent": {
+                    "type": "folder",
+                    "name": "2020-02-27",
+                    "id": "1234"
+                },
+                "owned_by": {
+                    "type": "user",
+                    "id": "12345",
+                    "name": "Jane Doe",
+                    "login": "email@example.com"
+                }
+            },
+            "created_by": {
+                "type": "user",
+                "id": "12345",
+                "name": "Jane Doe",
+                "login": "email@example.com"
+            },
+            "action_by": null,
+            "created_at": "2020-04-02T22:26:55-07:00",
+            "event_id": "66585c35-97a8-4882-9fec-ce7e178b2e53",
+            "event_type": "TASK_UPDATE",
+            "type": "event",
+            "additional_details": {
+                "size": 67801,
+                "task": {
+                         "isCompleted": false,
+                         "due": "2020-07-06T10:49:44-07:00"
+"id": "1234567"
+"description": "task description"
+"creatorId": "567890"
```

Comment:
This is not valid JSON. Please use 2-space indented JSON and make sure every section is properly alligned.